linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/HttpLoggerConfig.scala

```diff
@@ -0,0 +1,57 @@
+package io.buoyant.linkerd.protocol
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.{Filter, Service, ServiceFactory, Stack, Stackable}
+import com.twitter.finagle.http.{Request, Response}
+import com.twitter.util.Future
+import io.buoyant.config.PolymorphicConfig
+
+abstract class HttpLoggerConfig extends PolymorphicConfig { config =>
+
+  @JsonIgnore
+  def role: Stack.Role
+  @JsonIgnore
+  def description: String
+  @JsonIgnore
+  def parameters: Seq[Stack.Param[_]]
+
+  @JsonIgnore
+  def mk(params: Stack.Params): Filter[Request, Response, Request, Response]
+
+  def module = new Stack.Module[ServiceFactory[Request, Response]] {
+    override def role: Stack.Role = config.role
+    override def description: String = config.description
+    override def parameters: Seq[Stack.Param[_]] = config.parameters
+
+    override def make(
+      params: Stack.Params,
+      next: Stack[ServiceFactory[Request, Response]]
+    ): Stack[ServiceFactory[Request, Response]] = {
+      val filter = mk(params)
+      Stack.Leaf(role, filter.andThen(next.make(params)))
+    }
+  }
+}
+
+object HttpLoggerConfig {
+  val LoggerNilRole = Stack.Role("LoggerNil")
+  val LoggerNil = Stack.Leaf[ServiceFactory[Request, Response]](LoggerNilRole, null)
```

Comment:
Not a big deal but I realized after the fact that `com.twitter.finagle.stack.nilStack` exists and we could probably use that instead.